docs/de/user/installation.rst

```diff
@@ -86,7 +86,7 @@ Führe dieses Kommando aus, um das neueste Paket herunterzuladen und zu entpacke
 
 .. code-block:: bash
 
-    wget http://wllbg.org/latest-v2-package && tar xvf latest-v2-package
+    wget https://wllbg.org/latest-v2-package && tar xvf latest-v2-package
```

Comment:
@nicosomb the certificate doesn’t match the hostname, which will (hopefully) fail by default in all browsers and tools (this is what I put in the PR already)